src/integTest/resources/sample.springboot/test_spring_boot_apps_with_classifier_15.gradle

```diff
@@ -0,0 +1,44 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.19.RELEASE'
+	}
+	repositories {
+	   mavenLocal()
+		mavenCentral()
+	}
+	dependencies {
+		classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
+		classpath "net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:${lgpVersion}"
+	}
+}
+
+apply plugin: 'java'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'liberty'
+
+group = 'liberty.gradle'
+version = '1.0-SNAPSHOT'
+sourceCompatibility = 1.8
+
+bootRepackage {
+    mainClass = 'com.ibm.test.springboot.App'
+    classifier = 'test'
+}
+
+repositories {
+	mavenCentral()
+}
+dependencies {
+	compile("org.springframework.boot:spring-boot-starter-web:${springBootVersion}")
+	testCompile('org.springframework.boot:spring-boot-starter-test')
+}
+
+liberty {
+    server {
+    install.useOpenLiberty = false
```

Comment:
I think you can remove this now. Should work on WLP and OL right?

Reply:
We should add this if we can remove that line.
```
dependencies {
    libertyRuntime group: runtimeGroup, name: runtimeArtifactId, version: runtimeVersion
}
```

Reply:
done